src/skins/vector/css/matrix-react-sdk/structures/login/_Login.scss

```diff
@@ -44,6 +44,13 @@ limitations under the License.
     text-align: center;
 }
 
+.mx_Login_support {
+    text-align: center;
+    font-size: 13px;
+    margin-top: 0px;
+    opacity: 0.7;
```

Comment:
Ah ok - I've seen a few places where the CSS is using transparency apparently to achieve a particular colour rather than just making it that colour, but let's follow the current style for now.